ncm-network/src/main/pan/components/network/core-schema.pan

```diff
@@ -418,6 +418,8 @@ type structure_network = {
     "set_hwaddr" ? boolean
     "nmcontrolled" ? boolean
     "allow_nm" ? boolean
+    @{let Networkmanager manage the dns (nmstate). Default is false}
+    "nm_manage_dns" ? boolean
```

Comment:
Why not define the default in the schema, to match the comment. For me it would really be desirable to have the default set to false as otherwise you are at risk of breaking name resolution.

Reply:
the code sets to false if not defined, but yes can set it default to false. now done.
https://github.com/quattor/configuration-modules-core/pull/1601/commits/d748936ab0d238f0774f593a4928d821ff2f1255

Reply:
Thanks! I think it is a best practice in Pan, when there is a schema, to define the default value in the schema rather that in the code. It is both better exposed and easier to change. You can always add a catchall condition in the code to avoid producing errors in case something goes wrong with the schema but the expectation is that this part of the code should never been triggered!